api_app/tests_ma/test_api/test_routes/test_workspaces.py

```diff
@@ -272,32 +270,14 @@ async def test_get_workspace_by_id_get_returns_404_if_resource_is_not_found(self
         response = await client.get(app.url_path_for(strings.API_GET_WORKSPACE_BY_ID, workspace_id=WORKSPACE_ID))
         assert response.status_code == status.HTTP_404_NOT_FOUND
 
-    # [GET] /workspaces/{workspace_id}
-    @patch("api.dependencies.workspaces.WorkspaceRepository.get_workspace_by_id")
-    @patch("api.routes.workspaces.get_identity_role_assignments")
-    async def test_get_workspace_by_id_get_returns_422_if_workspace_id_is_not_a_uuid(self, access_service_mock, _, app, client):
-        access_service_mock.return_value = [RoleAssignment('ab123', 'ab124')]
-        response = await client.get(app.url_path_for(strings.API_GET_WORKSPACE_BY_ID, workspace_id="not_valid"))
-        assert response.status_code == status.HTTP_422_UNPROCESSABLE_ENTITY
-
-    @patch("api.dependencies.workspaces.WorkspaceRepository.get_workspace_by_id")
-    @patch("api.routes.workspaces.ResourceTemplateRepository.get_templates_information", return_value=[ResourceTemplateInformation(name="test")])
```

Comment:
Think this and the following tests are now removed (looking at locally), could you add them back

Reply:
cheers:
- the `422_` test has been moved elsewhere as requires a workspace user 
- the workspace service + user resource template GET tests have been moved into the ResearcherUser class as they now require a workspace user